NodeFirefoxDebug/Dockerfile.txt

```diff
@@ -1,55 +0,0 @@
-FROM selenium/node-firefox:3.4.0-chromium
```

Comment:
This file was in the repo, but not used anywhere. The `Dockerfile` comes from the NodeDebug folder.